Automattic/jetpack
# Add/memberships email to checkout form

Fixes 213-gh-automattic/gold

## Proposed changes:

* Adds a simple function to get the token from the cookie.
* Passes the token to the iframe to identify the user

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Apply D128616-code to your sandbox.
* Download this branch to your sandbox on top of D128616-code (answer 'n' to switching to trunk then 'y' to patch over D128616-code).
* Visit a test site which has both a newsletter tier / subscription and a paid content block (preferably, a paid content block with a different subscription plan, higher than one of the tiers).
* Open a private window and purchase the lowest tier subscription as a new subscriber.
* In the same private window, open a new tab and visit the post with the paid content block.
* Click "Subscribe" for the paid content block. You should a modal showing you your saved cards.
![Screenshot 2023-11-14 at 2 07 18 PM](https://github.com/Automattic/jetpack/assets/3662616/d92ce142-62a0-45a3-90d7-ede0c0e6ad6b)
* Click "New card".
* At the bottom of the form, your email address should be filled in automatically for the "Email" field.
